### PR TITLE
Include fix for failed pod setup

### PR DESCRIPTION
**Pull request by 3lvis**
Related to https://github.com/CocoaPods/CocoaPods/issues/3539

### CODE REVIEW

**Comment by orta** [comment]
This is a great idea @3lvis - :+1:

File: _posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown [hunk 1]
```diff
@@ -6,6 +6,8 @@ author: kyle
 categories: cocoapods
 ---
 
+_Note: If you got this problem after running `gem update`, [jump here](#user-content-i-got-linked-here-after-updating-my-gems-what-to-do) to find a solution_
```
Comment (segiddins) [inline _posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown]: period at the end of the sentence

File: _posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown [hunk 2]
```diff
@@ -106,6 +108,36 @@ adopters and pod providers. We are very thankful for your patience and
 understanding while we work towards a smoothly functioning version 1 of
 CocoaPods and its architecture.
 
+### I got linked here after updating my gems, what to do?
+
+This issue is being tracked down to an issue located in Psych. To fix it please use the following command:
+
+```shell
+rm -rf "~/Library/Caches/CocoaPods ~/.cocoapods;" \
```
Comment (segiddins) [inline _posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown]: let's get rid of the `\` ?

Comment (kylef) [inline _posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown]: Also, I don't think `;` shouldn't be inside the quotation.

File: _posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown (hunk 1 above)
Comment (3lvis) [inline _posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown]: @segiddins Added!

File: _posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown (hunk 2 above)
Comment (3lvis) [inline _posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown]: @kylef @segiddins The commands are chained, one runs after the other is finished.

screen shot 2015-12-09 at 21 35 35

Comment (kylef) [inline _posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown]: @3lvis I was pointing out that the following two are not identical:

```shell
$ echo "hello;" \
   echo "world"
hello; echo world
```

```shell
$ echo "hello"; \
   echo "world"
hello
world
```

You included the semicolon within the quotes so it's passed as an argument to `rm` and not as a command separator.

**Comment by 3lvis** [comment]
Can we get this merged? :)

**Comment by segiddins** [comment]
:ship: 
sorry, didn't see that the comments had been addressed!

**Pull request merged**